web/src/lib/domain-logic/starterKits.tsx

```diff
@@ -0,0 +1,103 @@
+import { getEssentialQuestion, hasEssentialQuestion } from "@/lib/domain-logic/essentialQuestions";
+import { modifyContent } from "@/lib/domain-logic/modifyContent";
+import { Industry } from "@businessnjgovnavigator/shared/industry";
+import { createEmptyProfileData, ProfileData } from "@businessnjgovnavigator/shared/profileData";
+
+export const insertRoadmapSteps = (content: string, roadMapsStepsLength: number): string => {
+  return modifyContent({
+    content,
+    condition: () => true,
+    modificationMap: {
+      numberRoadMapEasySteps: `${roadMapsStepsLength} Easy Step${roadMapsStepsLength > 1 ? "s" : ""}`,
+    },
+  });
+};
+
+export const insertIndustryContent = (content: string, industryId: string, industryName: string): string => {
+  return modifyContent({
+    content,
+    condition: () => industryId !== "generic",
+    modificationMap: {
+      industry: industryName,
+    },
+  });
+};
+
+export const createStarterKitProfileData = (industry: Industry): ProfileData => {
+  const emptyProfileData = createEmptyProfileData();
+  let newProfileData: ProfileData = {
+    ...emptyProfileData,
+    businessPersona: "STARTING",
+    industryId: industry.id,
+    legalStructureId: "limited-liability-company",
+  };
+
+  if (hasEssentialQuestion(industry.id)) {
+    const essentialQuestions = getEssentialQuestion(industry.id);
+    for (const essentialQuestion of essentialQuestions) {
+      // this only works if the values is defaults to a boolean, if its undefined by default as a value this will not catch it
+      if (typeof newProfileData[essentialQuestion.fieldName] === "boolean") {
```

Comment:
I'm not sure if there is a better way to capture the idea of this comment, or if it should just be deleted. 

It did take me a bit to understand that this was happening and I worry that someone in the future might fall into the same minor time wasting. 

Open to suggestions and alternative.

Reply:
maybe update the comment to say, essential questions with default boolean value.